auth/proxy/app.ts

```diff
@@ -39,20 +47,22 @@ export const lambdaHandler = async (event: APIGatewayProxyEventV2): Promise<APIG
       throw new Error('Missing Cognito URL parameter')
     }
 
-    const { headers, body, routeKey, rawQueryString } = event;
+    const { headers, body, rawQueryString, requestContext } = event;
 
     console.log('Calling auth proxy')
+    const { stage } = requestContext;
+    const { method, path } = requestContext.http;
 
-    const [method, path] = routeKey.split(' ')
+    const formattedPath = stripStageFromPath(stage, path);
```

Comment:
Without the removal of the stage prefix this causes the proxy to try forward to `cognito-endpoint/{stage}/oauth2/...` - can be removed if using custom domain name.